tests/test_messaging.py

```diff
@@ -817,6 +855,7 @@ def test_send(self):
         assert len(recorder) == 1
         assert recorder[0].method == 'POST'
         assert recorder[0].url == self._get_url('explicit-project-id')
+        assert recorder[0]._extra_kwargs['timeout'] == None
```

Comment:
Implement this check with `is None` to avoid the lint error: https://travis-ci.org/firebase/firebase-admin-python/jobs/351093644#L552